apps/partner2/test/index.coffee

```diff
@@ -10,7 +10,7 @@ Profile = require '../../../models/profile.coffee'
 newLayoutPartnerProfiles = _.map ['gallery_one', 'gallery_two', 'gallery_three'], (layout) ->
   new Profile fabricate 'partner_profile', owner: fabricate 'partner', profile_layout: layout
 
-deprecatedLayoutPartnerProfiles = _.map ['institution', 'gallery_default', 'gallery_deprecated'], (layout) ->
+deprecatedLayoutPartnerProfiles = _.map ['gallery_default', 'gallery_deprecated', 'institution'], (layout) ->
```

Comment:
Is this the line you were thinking of? https://github.com/artsy/force/pull/451/files#diff-1097aef15ab94b668ddc94cb07bc63edR11